### PR TITLE
Add specs for main class - Part 3

### PR DESCRIPTION
**Pull request by Phil-Friderici**
this commit finalizes the tests :)

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -433,20 +428,19 @@
     it { should contain_concat_file('/other/path') }
   end
 
-  context 'with channels_list set to valid string </other/path>' do
-    let(:params) { { :channels_list => '/other/path' } }
-
-    it { should contain_concat_file('/other/path') }
-  end
-
```
Comment (Phil-Friderici) [inline spec/classes/init_spec.rb]: Lines 436-440 was a doublette

File: spec/classes/init_spec.rb [hunk 2]
```diff
@@ -374,9 +371,7 @@
   context 'with use_default_logging_channel set to valid bool <false>' do
     let(:params) { { :use_default_logging_channel => false } }
 
-    # one bind::channel resources is valid because of hiera example data
-    it { should contain_bind__channel('from_hiera_common') }
-    it { should have_bind__channel_resource_count(1) }
+    it { should have_bind__channel_resource_count(0) }
```
Comment (Phil-Friderici) [inline spec/classes/init_spec.rb]: I have taken care that the hiera_merge examples do not pollute the other tests anymore.

**Comment by ghoneycutt** [comment]
Thanks!!

**Comment by Phil-Friderici** [comment]
btw: $slave_dir seems to be not used anywhere

**Comment by ghoneycutt** [comment]
Good catch. Created PR #15 to remove it

**Pull request merged**